lib/db.js

```diff
@@ -908,6 +908,14 @@ module.exports = (
     )
   }
 
+  DB.prototype.verifyTokensWithMethod = function (tokenId, tokenData) {
+    log.trace({op: 'DB.verifyTokensWithMethod', tokenId: tokenId})
```

Comment:
`, tokenId: tokenId})` --> `,tokenId, tokenData})` ?

Reply:
do you need `tokenData` as an object here? can this be just `verificationMethod` as a string?
plus it's not `tokenData` anymore I'm guessing? it's just an object with a verification method?